src/cli/src/consume/mod.rs

```diff
@@ -81,6 +81,13 @@ pub struct ConsumeOpt {
     /// Path to a SmartStream map wasm file
     #[structopt(long, group("smartstream"))]
     pub map: Option<PathBuf>,
+
+    /// Path to a WASM file for aggregation
+    #[structopt(long, group("smartstream"))]
+    pub aggregate: Option<PathBuf>,
+
+    #[structopt(long)]
+    pub accumulator: Option<PathBuf>,
```

Comment:
is this value or function? I believe this is initial value

Reply:
The `--aggregate` is the function, and `--accumulator` is the starting value

Reply:
`accumulator` implies it is sort of functional.  Suggest: `initial-value` or something like that.  For example, the rust fold use `init`:  https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.fold